### PR TITLE
Feature/wfdm 753

### PR DESCRIPTION
**Pull request by vivid-fxie**

### CODE REVIEW

**Comment by vivid-fxie** [comment]
Will need to use java 11 or 17 as lambda runtime in aws side in order for new open search client to compile

**Comment by vivid-fxie** [comment]
Will update this pr with java 17 as runtime

**Comment by vivid-fxie** [comment]
I was not able run lambda locally with java 17, using java 11 works. So I updated runtime to java 11.

File: wfdm-file-index-service/src/main/java/ca/bc/gov/nrs/wfdm/wfdm_file_index_service/OpenSearchRESTClient.java [hunk 1]
```diff
@@ -194,88 +182,67 @@ public IndexResponse addIndex(String content, String fileName, JSONObject fileDe
...
 		String json;
 		ObjectMapper mapper = new ObjectMapper();
 
 		try {
-			json = mapper.writeValueAsString(document);
+			json = mapper.writeValueAsString(searchDocumentResultsDto);
 		} catch (JsonProcessingException e) {
 			logger.error("json mapper failed :" +  e);
-			throw new ElasticsearchException("JSON?????", e);
+			throw new RuntimeException("JSON?????", e);
```
Comment (dhlevi) [inline wfdm-file-index-service/src/main/java/ca/bc/gov/nrs/wfdm/wfdm_file_index_service/OpenSearchRESTClient.java:199]: Not an appropriate Exception message. CLean it up to be clear on what error occured.

**Review by dhlevi — CHANGES_REQUESTED** [review]

File: wfdm-file-index-service/src/main/java/ca/bc/gov/nrs/wfdm/wfdm_file_index_service/OpenSearchRESTClient.java [hunk 2]
```diff
@@ -45,9 +34,9 @@ public class OpenSearchRESTClient {
 	// should likely be moved into a config file...
 	private static String serviceName = "es";
 	private static String region = "ca-central-1";
-	static RestHighLevelClient restClient;
-
-	static final AWSCredentialsProvider credentialsProvider = new DefaultAWSCredentialsProviderChain();
+	private static String units = "BKMGTPEZY";
```
[the remark this answers is not in the document]
Comment (vivid-fxie) [reply wfdm-file-index-service/src/main/java/ca/bc/gov/nrs/wfdm/wfdm_file_index_service/OpenSearchRESTClient.java:37]: It is used in the api before for converting file size back to bytes from human readable. It was hard coded before.

**Comment by dhlevi** [comment]
Looks like there's a conflict to clear up as well

**Pull request merged**